serverless/src/submitSwodlrOrder/constructOrderPayload.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+
+import { stringify } from 'qs'
+
+import { getEarthdataConfig } from '../../../sharedUtils/config'
+import { getClientId } from '../../../sharedUtils/getClientId'
+import { readCmrResults } from '../util/cmr/readCmrResults'
+
+export const constructOrderPayload = async ({
+  collectionConceptId,
+  earthdataEnvironment,
+  accessToken,
+  granuleParams
+}) => {
+  const newGranuleParams = { ...granuleParams }
+
+  delete newGranuleParams.json_data
```

Comment:
```suggestion
  // Remove data to use for CMR query
  delete newGranuleParams.json_data
```

Reply:
Just some kind of comment something different is fine its just kind of a weird line without context